_rules/SC1-2-1-video-only-audio-alternative

```diff
@@ -0,0 +1,108 @@
+---
+name: video only audio alternative
+description: |
+  Non-streaming `video` elements without audio must have an audio alternative
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+- Audio output
+- Visual output
+
+authors:
+- Brian Bors
+---
+
+## Test Procedure
+
+### Applicability
+
+The rule applies to any [non-streaming](#non-streaming) `video` element [visible on the page](#visible-on-the-page) where the video doesn't contains audio.
+
+### Expectation
+
+The visual information of each test target is available through an audio track.
+
+## Assumptions
+
+This rule assumes that a mechanism is available to start the video and that the video element is not simply used to display the poster.
+
+## Accessibility support
+
+There are no major accessibility support issues known for this rule.
+
+## Background
+
+- https://www.w3.org/TR/UNDERSTANDING-WCAG20/media-equiv-av-only-alt.html
```

Comment:
Same a before, add the [Title of the page](link to page)

Reply:
Fixed. Thanks.